src/predis/mock.clj

```diff
@@ -287,6 +290,19 @@
       (when (<= idx last-idx)
         (get vs idx'))))
 
+  (core/linsert [this k pos pivot v]
+    (assert (or (= pos "before") (= pos "after")) err-syntax)
```

Comment:
Looks like the [spec](http://redis.io/commands/linsert) has `pos` as upper case, so I'd prefer to accept values that way and then have something like

```clj
(let [pos' (string/lower-case pos)]
  (assert (#{"before" "after"} pos') err-syntax))
```

This is possible because sets can be used like functions in clj:

```clj
user=> (#{"one" "two"} "one")
"one"
user=> (#{"one" "two"} "missing-element")
nil
```